test/bdd/fixtures/testdata/create_activity.json

```diff
@@ -16,12 +16,12 @@
       "https://www.w3.org/ns/activitystreams",
       "https://trustbloc.github.io/did-method-orb/contexts/anchor/v1"
     ],
-    "id": "https://orb.domain1.com/transactions/bafkreihwsnuregceqh263vgdathcprnbvatyat6h6mu7ipjhhodcdbyhoy",
+    "id": "https://orb.domain1.com/transactions/QmbF6AKvzZ2fV7GubWq9v2P6qLMXyL6Y4ebZY8hrBng7QU",
```

Comment:
Had to update this to the actual CID that gets produced by our IPFS image (which is in v0 format instead of v1 like the example previously had) in order to get the BDD tests to pass. (I added code that ensures that the CID produced by the local CAS matches the one passed in to the resolver)